### PR TITLE
Shortcut keys

### PR DESCRIPTION
**Pull request by jsmey**
issue https://github.com/Raathigesh/majestic/issues/124

Added shortcut keys for run all tests, run selected test, open search and toggle watch

### CODE REVIEW

File: ui/hooks/use-keys.ts [hunk 1]
```diff
@@ -1,16 +1,37 @@
...
 
+export function hasKeys(expectedKeys: string[], pressedKeys: Map<String, boolean> ) {
+  return expectedKeys.every(k => pressedKeys.has(k));
+};
+
 export default function useKeys() {
   const [keys, setKeys] = useState(new Map());
+  const hotKeys = ["Control", "Enter", "Escape", "s", "t", "w"];
+
+  function copyKeyMap(map:Map<KeyboardEvent, boolean>) {
```
Comment (Raathigesh) [inline ui/hooks/use-keys.ts]: Instead of this function, we could do `const newMap = new Map(oldMap)`.

**Comment by Raathigesh** [comment]
This looks good to me. Hooks are so good for stuff like this. Thanks for sending the PR 💯 

I just left a single comment. Let me know what you think.

File: ui/hooks/use-keys.ts (hunk 1 above)
Comment (jsmey) [reply ui/hooks/use-keys.ts]: Good idea. I updated the PR

**Comment by Raathigesh** [comment]
Looks good. I'll try to make a release this weekend.

**Comment by M4cs** [comment]
Yayyyy!!! Great work guys. Been waiting for this!

**Pull request merged**